src/main/java/seedu/address/model/person/ModuleTagSet.java

```diff
@@ -24,34 +23,35 @@ public class ModuleTagSet implements Comparable<ModuleTagSet> {
      */
     private static final int DISPLAY_LIMIT = 10;
 
-    private final Set<ModuleTag> modules;
+    private final HashMap<String, ModuleTag> modules;
```

Comment:
Good way of integrating both the Lessons and the ModuleTag together, didn't think of this intuitive way to do it.